src/contribute/runner-guide.md

```diff
@@ -0,0 +1,1103 @@
+---
+layout: default
+title: "Runner Guide"
+permalink: /contribute/runner-guide/
+---
+
+# Runner Guide
+
+This document is aimed at someone who has a data processing system and wants
```

Comment:
on similar note, what about initial sentence at the start here...
This guide walks through how to implement a new runner. It is aimed at someone who has a data processing system and wants to use it to execute a Beam pipeline. The guide starts ...